polymath/extensions/item_ops.py

```
@@ -3,457 +3,473 @@
+        a1 = axis if axis >= 0 else axis + self._drank
+        k1 = self._ndims + self._nrank + a1
 
-    return obj
+        # Roll this axis to the beginning and slice it out
+        new_values = np.rollaxis(self._values, k1, 0)
+        new_values = new_values[index]
 
+        # Construct and cast
+        obj = Qube(new_values, self._mask, drank=self._drank - 1,
+                   example=self)
+        obj = obj.cast((type(self),) + classes)
```

Ruff (0.14.7)

Consider `(type(self), *classes)` instead of concatenation

Replace with `(type(self), *classes)`

(RUF005)